### DISCUSSION TITLE
Refoss Smart Energy Monitor, EM16

### DISCUSSION THREAD

**Discussion by CTJohnK** [krahabb/meross_lan#501] [General]
I know Refoss devices work with the Meross app and with meross_lan.  Has anyone tried the new Refoss Smart Energy Monitor, EM16 device with meross_lan?  Any reason why it might not work?

**Comment by krahabb** [comment]
Hello @CTJohnK , is it em16 or em06 ? I couldnt find that model online.
Anyway, a few weeks ago some support (for em06) was added and it was looking like working (see #476)

If this is only a question for the community I obviously cannot answer since I don't own any of those devices ;)

**Reply by CTJohnK** [reply]
EM16.
US Amazon link https://a.co/d/f1bsQZV
Refoss page https://refoss.net/collections/smart-energy-monitor

I presume it's built on the same platform as the EM06 as they are on the same page.

First question was to the community, second was to you. 🙂
